### ISSUE TITLE
network plugin cni failed to teardown pod: invalid version "": the version is empty

### ISSUE THREAD

**Issue by lidh15** [k3s-io/k3s#6005]
OS: Ubuntu
k3s: 1.24.2-k3s1
installed by airgap image

I googled it and they said we can edit flannel config map to assign the correct cni version
but there is no such config map in k3s
and how should I identify cni version? I mean I never installed it maunally, installation is automatically completed by k3s install.sh

**Comment by lidh15** [comment]
Update:
I updated those executeables in /opt/cni/bin to the latest version (both cni-plugins and cni-plugins-flannel) and restarted k3s, the situation is even worse, it failed at container creating with error 'network: unsupported CNI result version 1.0.0'
I searched through k3s  code base and found that k3s fixed 'cniVersion' to 1.0.0 at /etc/cni/net.d/10-flannel.conflist.
So I changed cni-plugins and cni-plugins-flannel to 1.0.0, but the error is still there!
I googled this error and it said that the error came from cri-o, what is that?
